src/lib/transformer.js

```diff
@@ -15,12 +15,12 @@ export default functions => {
             promises = Promise.all(promises);
 
         return then(promises, () => {
-            node[prop] = values.toString();  
+            node[prop] = values.toString();
         });
     }
 
     function transform(node) {
-        if (node.type !== 'function' || !functions.hasOwnProperty(node.value))
+        if (node.type !== 'function' || !Object.prototype.hasOwnProperty.call(functions, node.value))
```

Comment:
Any specific reason for this change?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

Doesn't _really_ apply in this case, but it's a "best practice" thing and the linter complained, so I succumbed 😄

Reply:
Fair enough, thanks!